micrometer-core/src/test/java/io/micrometer/core/instrument/binder/cache/HazelcastCacheMetricsTest.java

```diff
@@ -90,7 +90,7 @@ void reportMetrics() {
         assertThat(nearEvictions.value()).isEqualTo(nearCacheStats.getEvictions());
 
         // timings
-        TimeUnit timeUnit = TimeUnit.NANOSECONDS;
+        TimeUnit timeUnit = TimeUnit.MILLISECONDS;
```

Comment:
Thanks for the contribution! Can you explain why this change is needed? (I'm not particularly familiar with this unit test either)

Reply:
I think the previous tests are also incorrect, I tried with the old version but it was returning zero. But I know that `localMapStats.getTotalGetLatency()` returns the latency in milliseconds, not nanoseconds.